dll/shellext/zipfldr/resource.h

```diff
@@ -5,6 +5,9 @@
 #define IDB_HEADER                     200
 #define IDB_WATERMARK                  201
 
+/* Icons */
+#define IDI_ZIPFLDR 202
```

Comment:
Please use the same indent like the other numbers.

Reply:
Done.